lib/transformations.js

```diff
@@ -40,12 +48,14 @@ const header = ds => {
   }
 };
 
-const projection = (
-  // Dataset projection
-  meta, // projection metadata, example: ['Name'] or { toKey: [fromKey, funcs ]}
-  ds // array of records, example: [ { Id: 1, Name: 'Marcus' } ]
-  // Result: result array of records, example: [ { Name: 'Marcus' } ]
-) => {
+// Dataset projection
+//   meta - <string[]>, projection metadata | <Object>
+//   ds - <Object[]>
+// Returns: <Object[]>
+//
+// Example: complement(['Name'] || { toKey: [fromKey, funcs ]}, [ { Id: 1, Name: 'Marcus' } ]);
```

Comment:
I guess this is an example for another function? Also, do not mix several examples and if example with a single line is too long then split it to several lines.